smart_contracts/contracts/client/add-bid/src/main.rs

```diff
@@ -4,26 +4,17 @@
 extern crate alloc;
 
 use auction::DelegationRate;
-use casper_contract::{
-    contract_api::{account, runtime, system},
-    unwrap_or_revert::UnwrapOrRevert,
-};
-use casper_types::{auction, runtime_args, PublicKey, RuntimeArgs, URef, U512};
+use casper_contract::contract_api::{runtime, system};
+use casper_types::{auction, runtime_args, PublicKey, RuntimeArgs, U512};
```

Comment:
While we are here, can we put `auction::DelegationRate` in the `casper_types` stanza?